etc/openstack-config/openstack-config.yml

```diff
@@ -63,6 +63,37 @@
 # stackhpc.os-container-clusters role.
 #openstack_container_clusters_templates:
 
```

Comment:
Might be nice to drop a section partition here to separate from the actual cluster template config